### PR TITLE
Upgrade embedded Kotlin to 1.3.72

### PR DESCRIPTION
**Pull request by eskatos**
https://github.com/JetBrains/kotlin/releases/tag/v1.3.72

### CODE REVIEW

**Comment by eskatos** [comment]
@bot-gradle test RFN

File: subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/NamedDomainObjectCollectionExtensions.kt [hunk 1]
```diff
@@ -221,12 +221,21 @@ private constructor(
         fun <T> of(delegate: T) =
             ExistingDomainObjectDelegate(delegate)
     }
+
+    /**
+     * Gets the delegate value.
+     *
+     * @since 6.5
+     */
+    operator fun getValue(receiver: Any?, property: KProperty<*>): T =
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/NamedDomainObjectCollectionExtensions.kt:230]: ❌ The existing extension is just missing an `out` constraint on the generic argument, should be `ExistingDomainObjectDelegate<out T>...`

Comment (eskatos) [reply subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/NamedDomainObjectCollectionExtensions.kt]: Thanks! Fixed

**Comment by eskatos** [comment]
@bot-gradle test this

**Pull request merged**